pkg/harvester/l10n/en-us.yaml

```diff
@@ -300,9 +300,9 @@ harvester:
         minPages: Minimum Pages
         maxPages: Maximum Pages
         description:
-          boost: The number of scanned pages is incremented each time, and if the available memory is less than the ThresCoef.
-          decay: The number of scanned pages is decremented each time, and if the available memory is greater than the ThresCoef.
-          sleepMsec: The interval between each detection, is calculated by the formula for the interval between two scans.
+          boost: The number of scanned pages is incremented each time if the available memory is less than the ThresCoef.
```

Comment:
```suggestion
          boost: The number of scanned pages is incremented each time if the available memory is less than the threshold coefficients.
```

Reply:
Should I use the singular form `threshold coefficient` instead?